### PR TITLE
Complete AC.RB HW

### PR DESCRIPTION
**Pull request by tstwine**
I was at a 1 last night, now I'm at 4 since I came to the Office Hours.

### CODE REVIEW

File: class02/tina/ac.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+elsif  functional == "No" && current_temperature > desired_temperature
+	puts "Fix the A/C now! It's hot"
+
+else 
+	puts "Fix the A/C/ whenever you have the chance...It's cool..."
+	
+		
+	
+end
```
Comment (mobjohnson) [inline class02/tina/ac.rb]: Looks Good! Next time consider reducing the duplication of conditional statements using nested ifs

File: class02/tina/pyramid.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+puts "What character do you want to make the pyramid out of?"
+character = gets.chomp
+# ask the user for a character
+
+puts "How many rows of the #{character} in the pyramid?"
+row_count = gets.chomp.to_i
+
+character_count = 1cd
```
Comment (mobjohnson) [inline class02/tina/pyramid.rb]: 1cd does not work as it is neither a valid string or integer

File: class02/tina/pyramid.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+width = 100
+		
+
+ row_count.times do
+			puts (character * character_count).center(width)
+			character_count += 2
+	
+end
```
Comment (mobjohnson) [inline class02/tina/pyramid.rb]: this is close!

Comment (tstwine) [inline class02/tina/pyramid.rb]: Martin,
What is it that I'm omitting?
What can I do to make it go from "close" to " good or better"?

**Pull request merged**